coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -2727,3 +2727,60 @@ def tensor(context, node):
     shape = mb.shape(x=inputs[2], name=node.name+"_shape")
     context.add(mb.fill(shape=shape, value=val, name=node.name))
 
+@register_torch_op
+def reflection_pad2d(context, node):
+    inputs = _get_inputs(context, node)
+    x = inputs[0]
+    torch_pad = inputs[1].val
+    pad_flipped = torch_pad.reshape((-1, 2))[::-1].ravel()
+    pad = _np.pad(pad_flipped, (len(x.shape) * 2 - len(pad_flipped), 0))
+    context.add(mb.pad(x=x, pad=pad, mode='reflect'), node.name)
+
+
+@register_torch_op
+def reflection_pad1d(context, node):
+    inputs = _get_inputs(context, node)
+    x = inputs[0]
+    torch_pad = inputs[1].val
+    pad_flipped = torch_pad.reshape((-1, 2))[::-1].ravel()
+    pad = _np.pad(pad_flipped, (len(x.shape) * 2 - len(pad_flipped), 0))
+    context.add(mb.pad(x=x, pad=pad, mode='reflect'), node.name)
+
+
+@register_torch_op
+def replication_pad2d(context, node):
+    inputs = _get_inputs(context, node)
+    x = inputs[0]
+    torch_pad = inputs[1].val
+    pad_flipped = torch_pad.reshape((-1, 2))[::-1].ravel()
+    pad = _np.pad(pad_flipped, (len(x.shape) * 2 - len(pad_flipped), 0))
+    context.add(mb.pad(x=x, pad=pad, mode='replicate'), node.name)
+
+
+@register_torch_op
+def replication_pad1d(context, node):
+    inputs = _get_inputs(context, node)
+    x = inputs[0]
+    torch_pad = inputs[1].val
+    pad_flipped = torch_pad.reshape((-1, 2))[::-1].ravel()
+    pad = _np.pad(pad_flipped, (len(x.shape) * 2 - len(pad_flipped), 0))
+    context.add(mb.pad(x=x, pad=pad, mode='replicate'), node.name)
+
+@register_torch_op
+def constant_pad_nd(context, node):
```

Comment:
This op has been recently added in the last PR (here: https://github.com/apple/coremltools/blob/6245e289187c95d32b6ff65bc6348310ebbdf71a/coremltools/converters/mil/frontend/torch/ops.py#L912)

Can you update your PR by removing it or merging the two implementations if this covers more cases? 
We can keep the test!

Reply:
Oh, I didn't see that was implemented already, great. I removed my implementation.

Reply:
Cool, I merged your other PR on `log10`, once you resolve the conflict we can test and merge this one too.

Reply:
> Cool, I merged your other PR on `log10`, once you resolve the conflict we can test and merge this one too.

Done!